auth/src/main/java/com/firebase/ui/auth/ui/HelperActivityBase.java

```diff
@@ -58,6 +61,13 @@ public FlowParameters getFlowParams() {
         return mParams;
     }
 
+    @SuppressLint("all")
+    public void lockOrientation() {
+        if (getResources().getBoolean(R.bool.fui_portrait_orientation)) {
```

Comment:
We can remove the conditional here.  If the developer requests this through the `AuthUI` builder then we should always do it.